e2e/addon/addon_test.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	v1 "k8s.io/api/core/v1"
 
+	"github.com/Netflix/go-expect"
```

Comment:
why we need this package?

Reply:
When implementing the E2E test, I used the `sendline` function of this package to simulate user input


```go
It("Enable fluxcd-test-version whose version can't suit system requirements", func() {
	output, err := e2e.InteractiveExec("vela addon enable fluxcd-test-version", func(c *expect.Console) {
		_, err = c.SendLine("y")
		Expect(err).NotTo(HaveOccurred())
	})
	Expect(output).To(ContainSubstring("enabled successfully"))
	Expect(err).NotTo(HaveOccurred())
})
```

I find that in `e2e/commoncontext.go` uses this method, so I choose to use the same function.

Reply:
ok